engine/sql/061_pipeline_build_job_retry.sql

```diff
@@ -0,0 +1,6 @@
+-- +migrate Up
+ALTER TABLE workflow_node_run_job ADD COLUMN retry int DEFAULT 0;
+ALTER TABLE worker ADD COLUMN job_type text;
```

Comment:
should we update workflow_node_run_job to set the default value on existing tuples ?

Reply:
Yes we can !

Reply:
No, see PR comment below